avaframe/com4FlowPy/splitAndMerge.py

```diff
@@ -42,7 +42,8 @@ def tileRaster(fNameIn, fNameOut, dirName, xDim, yDim, U, isInit=False):
     #    os.makedirs(dirName)
 
     # largeRaster, largeHeader = iof.f_readASC(fNameIn, dType='float')
-    largeRaster, largeHeader = io.read_raster(fNameIn)
+    largeData = IOf.readRaster(fNameIn)
```

Comment:
I think we need to include `noDataToNan=False` here, otherwise we get problems with some existing code that is not able to handle `np.nan`

```suggestion
    largeData = IOf.readRaster(fNameIn, noDataToNan=False)
```

Reply:
see commit below